docs/topics/proc_exposing-application-metrics-using-prometheus-with-vertx.adoc

```diff
@@ -84,10 +85,12 @@ $ ./prometheus
 </dependency>
 ----
 
-. Collect metrics in your application using a custom class, for example:
+. Collect metrics in your application using a custom `launcher` class and override the `beforeStartingVertx` method to setup metrics options.
```

Comment:
I would say:

_As of 3.5.4, exposing metrics for Prometheus requires that you configure the Vert.x options in a custom `Launcher`  class. For example:_

Reply:
This will be fixed in my next commit.